README.md

```diff
@@ -1,46 +1,90 @@
+<div align="center">
+
+![Top banner](/docs/images/banner.jpg)
+
+AutoKitteh is a **developer** platform for workflow automation and
+orchestration. It is an easy-to-use, code-based alternative to no/low-code
+platforms (such as Zapier, Workato, Make.com, 8n8) with unlimited flexibility.
+
+In addition, it is a **durable execution** platform for long-running and
+reliable workflows. It is based on [Temporal](https://temporal.io/), hiding
+many of its infrastructure and coding complexities.
```

Comment:
```suggestion
AutoKitteh is a reliable **durable execution** platform for long-running workflows. It is based on [Temporal](https://temporal.io/), hiding
many of its infrastructure and coding complexities.
```

Reply:
Checked with Haim, he prefers to leave this as-is.